tests/scripts/all.sh

```diff
@@ -34,13 +34,26 @@ MEMORY=0
 SHORT=0
 FORCE=0
 
+OPENSSL="openssl"
+OPENSSL_LEGACY="$OPENSSL"
+GNUTLS_CLI="gnutls-cli"
+GNUTLS_SERV="gnutls-serv"
+GNUTLS_LEGACY_CLI="$GNUTLS_CLI"
+GNUTLS_LEGACY_SERV="$GNUTLS_SERV"
+
 usage()
 {
     echo "Usage: $0"
     echo -e "  -h|--help\t\tPrint this help."
     echo -e "  -m|--memory\t\tAdditional optional memory tests."
     echo -e "  -s|--short\t\tSubset of tests."
     echo -e "  -f|--force\t\tForce the tests to overwrite any modified files."
+    echo -e "     --openssl\t\tOpenSSL to use for most tests."
```

Comment:
To make it a bit clearer, could you indicate that for each OpenSSL and GNUTLS option, a parameter has to be supplied. e.g..
```
echo -e "     --openssl=<OpenSSL path>\t\tPath to OpenSSL executable to use for most tests."
```